src/cli.rs

```diff
@@ -1094,7 +1094,7 @@ pub fn handle_cli_edge_app_command(command: &EdgeAppCommands) {
 
             match edge_app_command.delete_app(&actual_app_id) {
                 Ok(()) => {
-                    info!("App deleted successfully.");
+                    info!("Edge App Deletion in Progress.\nRequest to delete the Edge App has been received and is now being processed. The deletion is marked for asynchronous handling, so it won't happen instantly.");
```

Comment:
@korvyashka  - don't use "info!" for user-facing messages. They will have a lot of technical data in the formatting list we don't want to show to users.